plugins/vercel/api_token.go

```diff
@@ -13,7 +13,7 @@ import (
 func APIToken() schema.CredentialType {
 	return schema.CredentialType{
 		Name:          credname.APIToken,
-		DocsURL:       sdk.URL("https://vercel.com/docs/cli/login"),
+		DocsURL:       sdk.URL("https://vercel.com/docs/rest-api"),
```

Comment:
Given the lengthiness of the page, it may be nice to use the fragment:
```suggestion
		DocsURL:       sdk.URL("https://vercel.com/docs/rest-api#authentication"),
```